tools/CopySchema/Program.cs

```diff
@@ -0,0 +1,202 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Security.Policy;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace CopySchema
+{
+    class Program
+    {
+        public class PackageInfo
+        {
+            public readonly string Name;
+            public readonly string Path;
+
+            public PackageInfo(string name, string path)
+            {
+                Name = name;
+                Path = path;
+            }
+
+            public override string ToString()
+            {
+                return $"[{Name}]: {Path}";
+            }
+
+            public override bool Equals(object obj)
+            {
+                if (obj == null || GetType() != obj.GetType())
+                {
+                    return false;
+                }
+
+                var other = (PackageInfo) obj;
+                return other.Name == Name && other.Path == Path;
+            }
+
+            public override int GetHashCode()
+            {
+                return Name.GetHashCode() * (31 + Path.GetHashCode());
+            }
+        }
+
+        public const string FROM_GDK_PACKAGES = "from_gdk_packages";
```

Comment:
Should be `FromGdkPackages`

Reply:
Are you sure? This is a directory that gets created in the root schema/ folder of the project, didn't think those folders were UpperCamelCase?

Reply:
Sorry, I meant the variable name, rather than the value

Reply:
Ah makes sense. Done.